packages/esm-patient-registration-app/src/patient-registration/field/id/id-field.component.tsx

```diff
@@ -98,7 +98,13 @@ export const Identifiers: React.FC = () => {
         });
       }
     }
-  }, [identifierTypes, setFieldValue, initializeIdentifier, defaultPatientIdentifierTypes, values.identifiers]);
+  }, [
+    identifierTypes,
+    setFieldValue,
+    defaultPatientIdentifierTypes,
+    values.identifiers,
+    initialFormValues.identifiers,
+  ]);
```

Comment:
I can see that `initializeIdentifier` is removed from the dependency and `initialFormValues.identifiers` is added.

Any reason for that?

Reply:
This is mainly for optimization by avoiding unnecessary re-renders of the component using the `initializeIdentifier` and the 
`values.identifiers`.

`initialFormValues.identifiers` ensure that the effect is triggered when the initial values of the identifiers field change. This could be important if the form is re-initialized with different initial values, for example, when editing an existing patient's record.

Reply:
**Urgent**
Please revert this for now. You can create a separate PR for this.
Thanks!